introduction/es/05-principles.asciidoc

```diff
@@ -0,0 +1,73 @@
+== Principios de InnerSource
+
+Cada compañía, equipo, project e individuo es diferente.
+Debido a esto, el camino a seguir y la manera de funcionar de InnerSource variará de una situación a otra.
+En su núcleo hay sin embargo cuatro principios que forman los cimientos de cualquier instancia de InnerSource.
+Estos principios se inspiran en los proyectos existosos de software libre y son requeridos en entornos InnerSource para alcanzar los beneficios que ya se han descrito.
+
+Estos principios son:
+* *Disponibilidad*
+* *Transparencia*
+* *Mentoría priorizada*
+* *Contribuciones voluntarias de código fuente*
+
+Echemos un vistazo a cada uno de estos principios en más detalle.
+
+
+=== Disponibilidad
```

Comment:
I'm not 100% sure about using this term. An alternative is "Apertura"... Ideas?

Reply:
"Acceso" or "Acceso abierto".

- "Disponibilidad" blurs into "Availability"
- "Accesibilidad" blurs into "Accesibility", as for physically handicaped / diverse functional people.

Reply:
Sounds good, I'll go for "Acceso abierto".